PGM/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -471,6 +477,11 @@ private void renderSidebar() {
                 }
             }
 
+            if (Config.Scoreboard.showIP()) {
+                rows.add("");
+                rows.add(ChatColor.AQUA + minecraftService.getLocalServer().datacenter() + Config.Scoreboard.networkIP());
```

Comment:
It would probably be better to not have the datacenter being added to the front, especially not without a period in between the datacenter and the rest of the ip.

Reply:
The period should be added to the IP in the config, idk how the datacenter works exactly but maybe the ip is just 'stratus.network' for example